jsonstar/JsonStar.Schema.fst

```diff
@@ -44,7 +44,7 @@ let string_of_number (x : number) =
     | Int i -> string_of_int i
 
 let mkSchemaEmpty (typ : schema_type) : Tot schema = 
-    Mkschema None None typ None None None []
+    Mkschema None None typ None None None None None []
```

Comment:
I think listing names of all the record fields would be a bit more readable.

Also, I recall there was some code above which was also full of `None`s, maybe it would get prettier if you made similar helpers for creating records will lots of optional fields set to `None`?